@navikt/core/react/src/timeline/TimelineRow.tsx

```diff
@@ -39,6 +40,7 @@ export const TimelineRow = forwardRef<HTMLOListElement, TimelineRowProps>(
   ({ label, className, headingTag = "h3", icon, ...rest }, ref) => {
     const { periods, id, active } = useRowContext();
     const { setActiveRow } = useTimelineContext();
+    const translate = useI18n("Timeline");
```

Comment:
Should we call `useI18n("Timeline")` once in Timeline.tsx instead, and just pass it down in TimelineContext?